src/main/kotlin/lotto/domain/LottoGame.kt

```diff
@@ -0,0 +1,5 @@
+package lotto.domain
+
+data class LottoGame(private val numbers: List<Int>) {
```

Comment:
```suggestion
data class LottoGame(val numbers: List<Int>) {
```
별도 getter 를 두지않고 바로 사용하면 되지 않을까요~? List 타입이라 외부에서 값을 수정할수 없으므로 사이드 이펙트가 없을 것 같아요!